### DISCUSSION TITLE
How to install on Fedora 33?

### DISCUSSION THREAD

**Discussion by benji88i** [Antynea/grub-btrfs#148] [Q&A]
Hi there,

This is not an issue as such but I was wondering how to install this on Fedora 33.

Also is there any documentation to learn how to set this up..

I am looking for an alternative to timeshift.

**Comment by Antynea** [comment]
hello,

>This is not an issue as such but I was wondering how to install this on Fedora 33.

You can install `grub-btrfs` like this:
```
git clone https://github.com/Antynea/grub-btrfs
cd grub-btrfs
sudo make install
```
Using fedora requires configuring `grub-btrfs`.
Edit `/etc/default/grub-btrfs/config`
uncomment and configure this 3 lines: 
(the supplied parameter should already work for fedora, **make sure that**.)
```
#GRUB_BTRFS_MKCONFIG=/usr/bin/grub2-mkconfig
#GRUB_BTRFS_BOOT_DIRNAME="/boot"
#GRUB_BTRFS_GRUB_DIRNAME="/boot/grub2"
```

Generate your Grub menu:
**Make sure the location of the grub.cfg file is correct**
`grub2-mkconfig -o /boot/grub/grub.cfg`

If you already had snapshots, they will appear in the Grub menu.

**Note**: Grub menu don't display by default on fedora 33.
To force display: 
`sudo grub2-editenv - unset menu_auto_hide` then Re-generate your Grub menu.
To redo: 
`sudo grub2-editenv - set menu_auto_hide=1` then Re-generate your Grub menu.
>Also is there any documentation to learn how to set this up..
https://github.com/Antynea/grub-btrfs#manual
https://github.com/Antynea/grub-btrfs/blob/master/config

I didn't create any official installation documentation.
